plugins/SitesManager/angularjs/sites-manager/sites-manager-site.controller.js

```diff
@@ -77,6 +87,16 @@
                 }, 'GET');
             }
 
+            var settings = $('.typeSettings fieldset').serializeArray();
```

Comment:
for now we use jQuery to get all the fields, we will change this later when we render the settings client side